pkg/riskanalysis/write_test_failure_summary.go

```diff
@@ -0,0 +1,84 @@
+package riskanalysis
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+)
+
+// WriteJobRunTestFailureSummary writes a more minimal json file summarizing a little info about the
+// job run, and what tests flaked and failed. (successful tests are omitted)
+// This is intended to be later submitted to sippy for a risk analysis of how unusual the
+// test failures were, but that final step is handled elsewhere.
+func WriteJobRunTestFailureSummary(artifactDir, timeSuffix string, finalSuiteResults *junitapi.JUnitTestSuite) error {
+
+	tests := map[string]*passFail{}
+
+	for _, testCase := range finalSuiteResults.TestCases {
+		if _, ok := tests[testCase.Name]; !ok {
+			tests[testCase.Name] = &passFail{}
+		}
+		if testCase.SkipMessage != nil {
+			continue
+		}
+
+		if testCase.FailureOutput != nil {
+			tests[testCase.Name].Failed = true
+		} else {
+			tests[testCase.Name].Passed = true
+		}
+	}
+
+	jr := ProwJobRun{
+		ProwJob: ProwJob{Name: os.Getenv("JOB_NAME")},
+		URL:     os.Getenv("JOB_URL"), // just a guess, may not exist
```

Comment:
In this [sample podinfo.json](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-nightly-4.12-e2e-aws-sdn-serial/1585015591855460352/podinfo.json) from this [prow job](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-nightly-4.12-e2e-aws-sdn-serial/1585015591855460352), there is no JOB_URL.  It does have BUILD_ID if you wanted to construct an URL later.

Reply:
Thanks, I will get this updated in my followup.